tests/result/test_qblox_result.py

```diff
@@ -263,18 +273,25 @@ def test_qblox_result_asymmetric_bins_raise_error(self, qblox_asymmetric_bins_re
         with pytest.raises(IndexError, match="Sequencers must have the same number of bins."):
             qblox_asymmetric_bins_result.counts_object()
 
-    def test_array_property_of_scope(self, dummy_qrm: DummyPulsar, qblox_result_scope: QbloxResult):
+    def test_array_property_of_scope(self, dummy_qrm: Pulsar, qblox_result_scope: QbloxResult):
         """Test the array property of the QbloxResult class."""
         array = qblox_result_scope.array
+        dummy_qrm.start_sequencer()
+        dummy_qrm.store_scope_acquisition(0, "single")
+        scope_data = dummy_qrm.get_acquisitions(0)["single"]["acquisition"]["scope"]
+        path0, path1 = scope_data["path0"]["data"], scope_data["path1"]["data"]
         assert np.shape(array) == (2, 16380)  # I/Q values of the whole scope
-        assert np.allclose(array, np.array([dummy_qrm._input_path0, dummy_qrm._input_path1]))
+        assert np.allclose(array, np.array([path0, path1]))
 
-    def test_array_property_of_binned_data(self, dummy_qrm: DummyPulsar, qblox_result_noscope: QbloxResult):
+    def test_array_property_of_binned_data(self, dummy_qrm: Pulsar, qblox_result_noscope: QbloxResult):
         """Test the array property of the QbloxResult class."""
         array = qblox_result_noscope.array
         assert np.shape(array) == (2, 1)  # (1 sequencer, I/Q, 1 bin)
-        path0, path1 = dummy_qrm._dummy_sequencers[0].demodulate(dummy_qrm._input_path0, dummy_qrm._input_path1)
-        assert np.allclose(array, np.array([[sum(path0[:1000])], [sum(path1[:1000])]]))
+        dummy_qrm.start_sequencer()
```

Comment:
Shouldn't this have an index too?

```suggestion
        dummy_qrm.start_sequencer(0)
```

Reply:
Good catch!!